Makefile

```diff
@@ -30,6 +30,21 @@ build:
 
 ##################################
 
+# Build multi-arch image
+
+PLATFORMS ?= linux/s390x,linux/amd64,linux/ppc64le
```

Comment:
And so far this works great, tested with an M1 mack and dockerx and it seems to be working just fine